Update DBM's Postgres troubleshooting (#15843)
DataDog/documentation: content/en/database_monitoring/setup_postgres/troubleshooting.md

#### Application is relying on search paths for specifying which schema to query

> Postgres does not expose the current [search path][20] in [`pg_stat_activity`][21], so it's not possible for the Datadog Agent to find out which search path is being used for any active Postgres processes. The work around for this limitation is to alter the search path for the user defined in the Postgres integration config to include the schema.

Reviewer: ```suggestion
Because Postgres does not expose the current [search path][20] in [`pg_stat_activity`][21], the Datadog Agent cannot find out which search path is being used for any active Postgres processes. The workaround for this limitation is to alter the search path for the user defined in the Postgres integration config to include the schema.
```

```sql

> ALTER ROLE datadog SET search_path = "$user",public,schema1,schema2,etc;

Reviewer 2: Nice, that seems like a decent workaround.

Reviewer 3: + 1
